lib/std/mem/Allocator.zig

```diff
@@ -308,7 +308,7 @@ pub fn resize(self: Allocator, allocation: anytype, new_len: usize) bool {
 /// In such case, it is more efficient for the caller to perform those
 /// operations.
 ///
-/// `allocation` may be an empty slice, in which case a new allocation is made.
+/// `allocation` may be an empty slice, in which case `null` is always returned.
```

Comment:
looks to be incorrect when new_len == 0.

Reply:
Oops, apologies, will fix shortly